_posts/sid_joshi/2017-02-16-JS-with-physical-world.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "JS + Physical World"
+breadcrumb: true
+author: sid_joshi
+date: 2017-02-16
+categories: sid_joshi
+tags:
+    - Javascript
+    - IoT
+teaser:
+    info: Javascript has been around for a while but in the past couple of years it has gained a lot of attention. Its about time we start using 
+        it for purposes other then web development. This article aims to demonstrate what can be achieved in the physical world by using Javascript. 
+        It will also point you in the direction of various tools, libraries, frameworks, devices and tutorials that can help you get started. 
+header: 
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+The Wikipedia defines Physical computing as building interactive systems by the use of software and hardware that can sense and respond to the analog world. 
+Physical computing includes IoT, robotics, wearables, interactive tangible medias and a lot more.  
+
+If you want to build a robot, you would require hardware, you need some sort of application/software to communicate with the hardware, you'll need controls to move the robot, and the robot needs to respond to 
+those controls respectively. But can we build something like this with JS? Yes! So, why should you use Javascript to start playing with microcontrollers? Its very easy to learn, has a huge community support and 
+learning it will also open doors for you into web application development. Also, because this article uses Javascript for the tutorial.
```

Comment:
`Also, because this article uses Javascript for the tutorial.` a little bit self-justifying? lol. Maybe say `Or, if you are a web developer and would like a low-barrier way to start playing with microcontrollers `.